lib/omniauth/login_dot_gov/authorization.rb

```diff
@@ -2,6 +2,7 @@ module OmniAuth
   module LoginDotGov
     class Authorization
       attr_reader :session, :client
+      VALID_AAL_VALUES = [2, 3, '3-hspd12']
```

Comment:
**suggestion**: what do you think about expecting a string here to keep the types consistent:
```suggestion
      VALID_AAL_VALUES = %w[2 3 3-hspd12].freeze
```

Reply:
Actually I think you needed to make this change since you're calling `.to_s` below.